components/automate-ui/src/app/helpers/auth/regex.ts

```diff
@@ -15,7 +15,7 @@ export class Regex {
     // Only allows wildcard alone or words and numbers, but not combined
     // Legal Values: *, chef, _state, etc.
     // Illegal Values: abc*, *chef, c*h*e*f, **
-    NO_MIXED_WILDCARD: '^(\\*|[-\\w]+)$'
+    NO_MIXED_WILDCARD: '^(\\*|[-!\\[\\]@#$%^&()+=/?><.,~\\w]+)$'
```

Comment:
I had a blind spot last go-around--this validation needs to match just what the back-end validation does:
```suggestion
    NO_MIXED_WILDCARD: '^(\\*|[^:*]+)$'
```
Reference: https://github.com/chef/automate/blob/5b88f9da5daeeb9ec58016a6f9c13f52444703cd/api/interservice/authz/v2/authz.proto#L63

Also, this raises a small UX question, @susanev:
The error pops up to say "Name must be text or a wildcard but not both.".
But in light of this change you might want to revise.
"Name may not contain colons or wildcards"
... except that is a bad message if the user types "**", which is also not allowed, so possibly:
"Use either a single wildcard or a name without colons or wildcards."
That is very precise now, but probably not the greatest wording.

Reply:
Thanks for this @msorens!  Looks like it may be easier to use two different Regex's.  One for LDAP/SAML and the other for token ID and locals.  I'll swap this Regex out for the former.